.travis.yml

```diff
@@ -13,7 +13,7 @@ before_script: >
   node_version=$(node -v);
   if [ ${node_version:3:1} = "." ]; then
     echo "Node 10+"
-    npm install --no-save "eslint@6"
+    npm install
```

Comment:
Won't this just case `npm install` to happen twice on node 10+? It is already run by travis by default.

Reply:
Ah, ok, I've added a commit to remove.